packages/@secretlint/secretlint-rule-no-dotenv/package.json

```diff
@@ -0,0 +1,60 @@
+{
+    "name": "@secretlint/secretlint-rule-no-dotenv",
+    "version": "0.1.0",
+    "description": "A secretlint rule for dotenv",
+    "keywords": [
+        "secretlint",
+        "rule",
+        "dotenv"
+    ],
+    "homepage": "https://github.com/secretlint/secretlint/tree/master/packages/@secretlint/secretlint-rule-no-dotenv/",
+    "bugs": {
+        "url": "https://github.com/secretlint/secretlint/issues"
+    },
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/secretlint/secretlint.git"
+    },
+    "files": [
```

Comment:
Add author and license https://github.com/secretlint/secretlint/blob/44e1388cbead71f09b2cb81352429002dac53a8c/packages/%40secretlint/secretlint-rule-secp256k1-privatekey/package.json#L23-L24

Reply:
@azu resolved by follows:

- 2b19e36ba10ce57034ae8ca939ea61d5efd8ca0a
- 63d9d7bc77c44e1e43c6767b0cd85f8a191c2e5e